rawfile/rawfile_servicer.py

```diff
@@ -208,14 +223,19 @@ def CreateVolume(self, request, context):
                 grpc.StatusCode.INVALID_ARGUMENT, "Topology key not found... why?"
             )
 
-        params = normalize_parameters(request.parameters)
-        thin_provision = params.get("thinprovision", "no")
         format_options = params.get("formatoptions", "").strip()
+        snap_inuse = str_to_bool(params.get("snapinuse", "no"))
+        freezefs = str_to_bool(params.get("freezefs", "no"))
```

Comment:
this should come from the snapshot class rather than volume?

Reply:
I have used storage class parameters because we will need them outside of the snapshots

Reply:
Ah ok, where else would it be used then?

Reply:
Shouldn't default behavior be safety first, a consistent snapshot? Other user may have nasty surprise later when trying to use it
So I think `dangerous` should be an opt-in

Reply:
Yeah

any updates on product document itself?